### PR TITLE
Update pull_request_template.md

### PR DESCRIPTION
**Pull request by kashimiz**
Adding diagnostic events reminder to checklist.

### Pull request checklist

* [X] My changes **do not** require documentation changes
    * [ ] Otherwise: Documentation issue linked to PR
* [X] My changes **should not** be added to the release notes for the next release
    * [ ] Otherwise: I've added my notes to `release_notes.md`
* [X] My changes **do not** need to be backported to a previous version
    * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [X] I have added all required tests (Unit tests, E2E tests)

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ resolves #issue_for_this_pr
 * [ ] My changes **do not** need to be backported to a previous version
     * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
 * [ ] I have added all required tests (Unit tests, E2E tests)
+* [ ] I have added/updated all related diagnostic events and their documentation
+    * [ ] If so: Documentation issue linked to PR
```
Comment (soninaren) [inline .github/pull_request_template.md]: Should we also add an option that says `i don't have any events to log.` ?

Comment (fabiocav) [reply .github/pull_request_template.md]: The change can follow the pattern established here: 
- [ ]   not needed
  - [ ]  Otherwise: Link/info

Comment (kashimiz) [reply .github/pull_request_template.md]: Updated.

**Review by soninaren — APPROVED** [review]

**Review by liliankasem — APPROVED** [review]

**Comment by liliankasem** [comment]
@kashimiz @soninaren should we backport this to v3 branch or are diagnostic events only in v4?

**Comment by soninaren** [comment]
> @kashimiz @soninaren should we backport this to v3 branch or are diagnostic events only in v4?

This should be backported to v3 for sure. Diagnostic events is in v3 as well

**Pull request merged**